frontend/vue/components/CodeExercise/ExerciseActionsBar.vue

```diff
@@ -16,7 +16,8 @@
     </button>
     <div v-if="isRunning" class="exercise-actions-bar__running-indicator">
       <bx-loading class="exercise-actions-bar__running-indicator__icon" assistive-text="Running" type="small" />
-      <span class="exercise-actions-bar__running-indicator__label">{{ $translate('Running') }}</span>
+      <span v-if="isGrading" class="exercise-actions-bar__running-indicator__label">{{ $translate('Grading') }}</span>
```

Comment:
only question here:

since we now have a _grading_ indicator, should this `running-indicator` class be reduced to just `indicator`?

💭

Reply:
Good catch @techtolentino , I would add that the background for the `grading` status is black. Should we maintain the color to differentiate both "actions"?

Reply:
thanks @techtolentino @Tansito! good points on both. i have gone ahead and [updated class name and background color](https://github.com/Qiskit/platypus/pull/1425/commits/205badc1bfdf71ca08e1a3533f7645b9c9fc96fc). check out the updated video above or the preview link (when finishes building)

Reply:
This is great @vabarbosa 

nicely done!